src/url-loader/package-url-resolver.ts

```diff
@@ -57,10 +57,12 @@ export class PackageUrlResolver extends UrlResolver {
   }
 
   resolve(
-      unresolvedHref: FileRelativeUrl|PackageRelativeUrl,
-      baseUrl: ResolvedUrl = this.packageUrl,
+      firstHref: ResolvedUrl|PackageRelativeUrl, secondHref?: FileRelativeUrl,
       _import?: ScannedImport): ResolvedUrl|undefined {
-    const resolvedHref = this.simpleUrlResolve(unresolvedHref, baseUrl);
+    const [baseUrl, unresolvedHref] = typeof secondHref === 'undefined' ?
```

Comment:
Prefer `secondHref === undefined`, it's likely to be much faster, and it's what we do everywhere else

Reply:
Ok.  This `typeof x === 'undefined'` of mine is an old habit.  Will switch!